src/Model.cc

```diff
@@ -687,7 +739,16 @@ void Model::SetPoseRelativeToGraph(sdf::ScopedGraph<PoseRelativeToGraph> _graph)
   }
   for (auto &ifaceModelPair : this->dataPtr->interfaceModels)
   {
-    ifaceModelPair.second->InvokeRespostureFunction(childPoseGraph);
+    // Don't invoke reposture for interface models that were merged.
```

Comment:
nit Unclear why we don't do this. Consider explaining briefly?

e.g. "... because we will reposture them below"

Reply:
Done in https://github.com/ignitionrobotics/sdformat/pull/768/commits/1907e5de87c02c1d38d8347030b8d0e177a9efca